web/shared/src/typing_status.ts

```diff
@@ -1,6 +1,18 @@
 import _ from "lodash";
 import assert from "minimalistic-assert";
 
+type TypingStatusWorker = {
+    get_current_time: () => number;
+    notify_server_start: (recipient: number[]) => void;
+    notify_server_stop: (recipient: number[]) => void;
```

Comment:
`user_ids_array` would be a clearer representation of the parameters.

Reply:
Hmm, looks like we are using the term `recipient` in other places in this file when it's actually an array of user ids.
We use `user_ids_array` in `typing.js`, and name the parameter type as `RecipientUserIds` in `typing_status.js.flow`.

At some point, this naming might have been confusing as we have test cases that use `"alice"` for `current_recipient`. It would be helpful if we can be consistent with `recipient_ids` in another refactoring commit.